app/src/main/res/values/strings.xml

```diff
@@ -686,13 +686,34 @@
     <string name="qualities">Qualities</string>
     <string name="profile_background_des">Profile background</string>
     <string name="unable_to_inflate">UI was unable to be created correctly, this is a MAJOR BUG and should be reported immediately %s</string>
-
-
-    <string name="tv_no_focus_tag" translatable="false">tv_no_focus_tag</string>
     <string name="already_voted">You have already voted</string>
+
     <string name="favorites_list_name">Favorites</string>
     <string name="favorite_added">%s added to favorites</string>
     <string name="favorite_removed">%s removed from favorites</string>
     <string name="action_add_to_favorites">Add to favorites</string>
     <string name="action_remove_from_favorites">Remove from favorites</string>
+
+    <string name="duplicate_title">Potential Duplicate Found</string>
+    <string name="duplicate_add">Add</string>
+    <string name="duplicate_replace">Replace</string>
+    <string name="duplicate_cancel" translatable="false">@string/sort_cancel</string>
+    <string name="duplicate_message_bookmarks">
```

Comment:
3 nearly identical messages makes no sense, it takes a lot longer to translate that into like 50 languages. Keep it to 1 and use %s to fill the type, that way we can add this to future stuff and keep it easier to translate

Reply:
The reason I kept it separate was the subtle differences between each of the 3, in particular subscriptions, originally had it as a combined message but depends on the action it didn't make to much sense to be combined, if you want I can go back to that again though and figure something out.

Reply:
I went ahead and did this now, though it still has two: one for single duplicate found, and one for multiple